builtin/providers/aws/config.go

```diff
@@ -12,8 +12,10 @@ import (
 	"github.com/mitchellh/goamz/ec2"
 	"github.com/mitchellh/goamz/elb"
 	"github.com/mitchellh/goamz/rds"
-	"github.com/mitchellh/goamz/route53"
 	"github.com/mitchellh/goamz/s3"
+
+	awsGo "github.com/awslabs/aws-sdk-go/aws"
```

Comment:
`goamz/aws` is still present and used here (`aws.GetAuth()`, probably a few others). I figured we'd port the resources over, and at the end remove `goamz/aws`. 

I can do that earlier in the process though if we think we should. Here though, I was touching as little as possible trying not break things.